### PR TITLE
Change attributes for XOR EFB and OffPage connectors

### PR DESCRIPTION
**Pull request by AlexTxen**
Added ControlNode atrribute to SignalOffPage and SignalOnPage classes, ref. #17 
Changed Notation attribute to '≠' for "XOR" EFB, ref #23 
Changelog prepared for 0.0.9 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New SystemUnitClass with Name="ContextHiding" under DocumentClassLibrary
 - New SystemUnitClassLib with Name="ContextHidingElementLibrary", with members:
   - Context, Transition
+- Added ControlNode atrribute to SignalOffPage and SignalOnPage classes
+- Changed Notation attribute to '≠' for "XOR" EFB
```
Comment (Erik0x42) [inline CHANGELOG.md:25]:
```suggestion

### Changed

- Changed Notation attribute to '≠' for "XOR" EFB 
```

**Review by Erik0x42 — COMMENTED** [review]
I believe it has been discussed to give the next release after 0.0.8 the version number "1.0.0", due to major changes, has this discussion been concluded?

**Comment by Erik0x42** [comment]
The date in the changelog (latest release heading) implies that the next release will be published today.
I would suggest that issues are mapped to planned releases so that we have a visual indicator of readiness/completeness for a specific release.
See #32

**Comment by cdenisey** [comment]
> I believe it has been discussed to give the next release after 0.0.8 the version number "1.0.0", due to major changes, has this discussion been concluded?

Erik, we discussed in the initial workshop that the AML version "1.0.0" will be released when the IEC63131 edition 2.0 is officially issued to integrate all changes related to IEC standard updates.

**Review by cdenisey — APPROVED** [review]

**Pull request merged**